### ISSUE TITLE
Chaos Monkey for Spring boot not working with java11

### ISSUE THREAD

**Issue by maiksensi** [codecentric/chaos-monkey-spring-boot#86]
Seems chaos-monkey-for-spring-boot is currently not working with java11. We should change that at some point. (Guess I would mark this as a feature request)

Basically two tests are failing:
```
Results :

Failed tests:   allowInterruptionOfAssaultDuringHoldPeriod(de.codecentric.spring.boot.chaos.monkey.assaults.MemoryAssaultIntegrationTest): Memory did not reach half exhaustion before timeout

Tests in error: 
  initializationError(de.codecentric.spring.boot.chaos.monkey.component.ChaosMonkeySchedulerTest): java.lang.reflect.InvocationTargetException
  initializationError(de.codecentric.spring.boot.chaos.monkey.assaults.LatencyAssaultTest): java.lang.reflect.InvocationTargetException

Tests run: 87, Failures: 1, Errors: 2, Skipped: 0
```

How to reproduce: 
- use java11 locally and run `mvn test`

**Comment by WtfJoke** [comment]
The InvocationTargetException has something to do with the used PowerMock/Mockito-Version (probably the one used is not java9+ compatible)

**Comment by WtfJoke** [comment]
Closed by #87
